src/tests/ftest/datamover/dm_large_file.yaml

```
@@ -44,7 +44,7 @@ ior:
       block_size: '5G'        # aggregate of 150G for dcp and 50G for fs_copy
   objectclass:
     SX:
-      dfs_oclass: "SX"
+      dfs_oclass: "EC_4P1G1"
```

The test is failing because of no space so try with GX instead of G1,
```suggestion
      dfs_oclass: "EC_4P1GX"
```

---

Yes, I think we should use GX instead of G1.

---

Fixed